.github/workflows/unit_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+
+  test:
+    name: Tests
+    uses: theforeman/actions/.github/workflows/test-gem.yml@v0
+    with:
+      command: bundle exec rake test
```

Comment:
Trailing new line. As Ewoud said, check your editor to automatically add new line at the end

Reply:
I think we need to decide when we want to insert a new line and when it is unnecessary. See https://github.com/theforeman/hammer-cli-foreman-ansible/pull/27#discussion_r1496063705.

Reply:
> I think we need to decide when we want to insert a new line and when it is unnecessary.

In UNIX files every line has a line ending, but in that case you had an additional empty line. Most editors have a setting to enforce this and many do that by default.